storage-account.tf

```diff
@@ -1,5 +1,83 @@
 // Shared and specialised Storage Accounts
 
+locals {
+  sa_subnets = [
+    data.azurerm_subnet.jenkins_subnet.id,
+    data.azurerm_subnet.aks-00-mgmt.id,
+    data.azurerm_subnet.aks-01-mgmt.id,
+    data.azurerm_subnet.aks-00-infra.id,
+    data.azurerm_subnet.aks-01-infra.id,
+    data.azurerm_subnet.aks-00-preview.id,
+    data.azurerm_subnet.aks-01-preview.id
+  ]
+}
+
+data "azurerm_virtual_network" "mgmt_vnet" {
+  provider            = azurerm.mgmt
+  name                = local.mgmt_network_name
+  resource_group_name = local.mgmt_network_rg_name
```

Comment:
where are these locals defined?

Reply:
yes I couldn't get sandbox working to validate, got it working now ;)